core/templates/dev/head/pages/exploration_editor/simple_editor_tab/SimpleEditorShimServiceSpec.js

```diff
@@ -0,0 +1,210 @@
+// Copyright 2014 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for the SimpleEditorShimService.js
+ */
+describe('Simple Editor Shim Service', function() {
+  var currentStateName;
+  var srv;
+  var titleMemento;
+  var states;
+  var stubs;
+
+  beforeEach(function() {
+    currentStateName = 'state1';
+    srv = null;
+    titleMemento = 'SampleTitle';
+    states = {
+      state1: {
+        name: 'state1',
+        content: [
```

Comment:
Optional: it might be more compact to write:

    content: [{
      value: '...'
    }, {
      value: '...'
    }, {
      value: '...'
    }]